src/app/common/utils.ts

```diff
@@ -270,6 +270,10 @@ export function isString(value: unknown): value is string {
   return typeof value === 'string';
 }
 
+export function isUndefined(value: unknown) {
```

Comment:
Shouldn't we add ```:value is undefined ``` to inform typescript ?

Reply:
Good spot